sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/SchemaPruning.scala

```diff
@@ -120,7 +120,8 @@ object SchemaPruning {
   private def getRootFields(expr: Expression): Seq[RootField] = {
```

Comment:
How about `private` -> `private[expressions]` if there is no other way to add tests?